### ISSUE TITLE
[feature request] Run calibration for filament profile on the machine or bambu handy.

### ISSUE THREAD

**Issue by BambuLabDiscord** [bambulab/Bambu-Handy#124]
I have my printers in a small workspace downstairs, while my bambu studio pc is upstairs. If I am to profile a filament I have to down to load the filament, then upstairs to start profile and wait 15-17min before it is done. Then go downstairs to scrape off for next profile. 
If I was able to either do this on bambu handy or printer display it would be much easier since I could start on all three printers and do something else while they work on the profile. (And please have it as option to run both flow and pressure in one procedure instead of two)

**Comment by GuzziRaz** [comment]
Agreed. Also, Bambu Handy does not show K values nor lets us select from saved ones. It should at the very least show what’s used. 

![IMG_5598](https://github.com/bambulab/Bambu-Handy/assets/143495653/c8d39fef-c224-4fe6-a129-7d18db000f2d)

![Screenshot 2023-09-28 at 21 42 23](https://github.com/bambulab/Bambu-Handy/assets/143495653/6139a93d-622e-46eb-9040-7de06ddf140b)
